docs/topics/high-performance-computing.md

```diff
@@ -260,6 +260,8 @@ There are many workload managers offered in the [Azure Marketplace](https://azur
 
 SaaS providers or developers can use the Batch SDKs and tools to integrate HPC applications or container workloads with Azure, stage data to Azure, and build job execution pipelines.
 
+![Azure Batch Architecture Example](images/az-batch-arch01.png "Azure Batch Architecture Example")
```

Comment:
The images don't really seem like they are adding a lot of value.  They are not architecture examples and look more like marketing content.  Can you explain why we need these in architecture content?

If you do keep them, not only would we need to fix the transparency issue already identified on this thread, I'd suggest removing the "title" text from the image -- it's not accessible and isn't doing anything that the surrounding text cannot do.